apps/sms/views/conversation/test/unit/compose_test.js

```diff
@@ -1190,6 +1187,9 @@ suite('compose_test.js', function() {
   ['image', 'video', 'audio', 'other'].forEach(function(type) {
     suite(type + ' attachment pre-send menu display', function() {
       setup(function() {
+        loadBodyHTML('/index.html');
```

Comment:
__question:__ as far as I understand tests don't work with ```loadBodyHTML```, so just curious what is the reason?